ProcessLib/SmallDeformationWithLIE/SmallDeformationProcessData.h

```diff
@@ -0,0 +1,79 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef PROCESSLIB_SMALLDEFORMATION_WITH_LIE_SMALLDEFORMATIONPROCESSDATA_H_
+#define PROCESSLIB_SMALLDEFORMATION_WITH_LIE_SMALLDEFORMATIONPROCESSDATA_H_
+
+#include <memory>
+
+#include "MeshLib/PropertyVector.h"
+
+#include "MaterialLib/SolidModels/MechanicsBase.h"
+#include "MaterialLib/FractureModels/FractureModelBase.h"
+
+#include "ProcessLib/SmallDeformationWithLIE/Common/FractureProperty.h"
+
+namespace MeshLib
+{
+class Element;
+}
+
+namespace ProcessLib
+{
+namespace SmallDeformationWithLIE
+{
+template <int DisplacementDim>
+struct SmallDeformationProcessData
+{
+    SmallDeformationProcessData(
+        std::unique_ptr<MaterialLib::Solids::MechanicsBase<DisplacementDim>>&& material,
+        std::unique_ptr<MaterialLib::Fracture::FractureModelBase<DisplacementDim>>&& fracture_model,
+        std::unique_ptr<FractureProperty>&& fracture_prop
+        )
+        : _material{std::move(material)}, _fracture_model{std::move(fracture_model)},
+          _fracture_property{std::move(fracture_prop)}
+    {
+    }
+
+    SmallDeformationProcessData(SmallDeformationProcessData&& other)
```

Comment:
It doesn't compile with this (https://svn.ufz.de:8443/job/OGS-6/job/Linux-PRs-dynamic/1394/console). I would keep as it is